modules/support_ticket/support_ticket.module

```diff
@@ -266,6 +266,54 @@ function support_ticket_load($nid = NULL, $reset = FALSE) {
 }
 
 /**
+ * Determines what fields have changed when the support ticket is updated.
+ *
+ * @param object $support_ticket
+ *   The support ticket entity object.
+ */
+function support_ticket_entity_update($support_ticket) {
+  $original = $support_ticket->original;
+  $fields_original = $support_ticket->original->getFields();
+  $fields_updated = $support_ticket->getFields();
+
+  $comment = '';
+
+  // Don't log changes to the following fields when the support_ticket is updated.
+  // @todo: "field_ticket_update" is a configurable comment field; I think we can simply
+  //        ignore any comment field.
+  $ignore = array('stid', 'uuid', 'vid', 'uid', 'langcode', 'changed', 'revision_timestamp', 'revision_uid', 'revision_log', 'revision_translation_affected', 'field_ticket_update');
+  foreach ($fields_updated as $name => $field) {
+    if (!in_array($name, $ignore)) {
+      $updated_value = $field->getValue();
+      $original_value = isset($fields_original[$name]) ? $fields_original[$name]->getValue() : FALSE;
```

Comment:
You could use array_diff_key() to filter out the exclude quicker - would save some indentation.

I'm not sure the comparison here is enough to determine whether the field really changed or not. Could possibly borrow from: https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Entity%21ContentEntityStorageBase.php/function/ContentEntityStorageBase%3A%3AhasFieldValueChanged/8

With the value comparison, I'd consider actually rendering the field values rather than using the values directly in the message - would for example allow images to be rendered side by side if different.